packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -132,9 +132,26 @@ abstract class CreateBase extends FlutterCommand {
     );
   }
 
+  /// Pattern for a Windows file system drive (e.g. "D:").
+  ///
+  /// `dart:io` does not recognize strings matching this pattern as absolute
+  /// paths, as they have no top level back-slash; however, users often specify
+  /// this
+  static final RegExp _kWindowsDrivePattern = RegExp(r'^[A-Z]:$');
```

Comment:
Is there anything helpful in `package:path` instead of potentially duplicating this logic?

Reply:
I see [`rootPattern`](https://github.com/dart-lang/path/blob/master/lib/src/style/windows.dart#L29), but that is a superset of what we want here, including a trailing slash (this is the same pattern that dart:io uses).

However, from looking at that, I can see that this regex should include lower-case letters.

Reply:
oh wait, never mind, this helper looks like it does the trick: https://github.com/dart-lang/path/blob/master/lib/src/utils.dart#L18

Reply:
I saw that, but couldn't tell if it was public.

Reply:
Shucks, it isn't exported